tests/e2e/docs/supervisor_cluster_setup.md

```diff
@@ -77,8 +77,11 @@ datacenters should be comma separated if deployed on multi-datacenters
     export CLUSTER_FLAVOR="WORKLOAD"
     export GINKGO_FOCUS="csi-supervisor"
     export SVC_NAMESPACE="e2e-test-namespace"
+    export SVC_NAMESPACE_TO_DELETE="e2e-namespace-to-delete"
```

Comment:
Can you also update the document to mention that this namespace needs to be created in advance? Probably in the section where it mentions the creation of SVC_NAMESPACE.

Reply:
Updated